scripts/nz-buildings-load

```diff
@@ -0,0 +1,204 @@
+#!/bin/sh
```

Comment:
This shouldn't be committed, right? Just the `.in`?

Reply:
Ah, a few lines were deleted from the `.gitignore` - I think those need to be added back in with adjustments for new file locations.